MAXQDA-Conversion/req.txt

```diff
@@ -0,0 +1,3 @@
+pandas=0.23.3=py36h6440ff4_0
+numpy=1.14.5=py36h648b28d_4
+numpy-base=1.14.5=py36ha9ae307_4
```

Comment:
I get error messages when I try to install this:
```Invalid requirement: 'pandas=0.23.3=py36h6440ff4_0'```

Also, let's call the file "requirements.txt".